### ISSUE TITLE
Flagging collection dates for `PP_000JSDD.3` and `PP_000JSEB.3`

### ISSUE THREAD

**Issue by j23414** [pathoplexus/curation_reports#3]
### Describe the possible issue

NCBI GenBank records [EF631122](https://www.ncbi.nlm.nih.gov/nuccore/EF631122) and [EF631123](https://www.ncbi.nlm.nih.gov/nuccore/EF631123) (corresponding to PP_000JSDD.3 and PP_000JSEB.3) have collection date annotations of "05-Aug-1931" and "05-Aug-1931" in location "USA: Illinois, Cook County". This is inconsistent with known history of West Nile Virus (WNV) in the USA, as the first sequenced case should be around or after 1999.

### Evidence of the problem

Searching the literature of the submitters:

`EF631123` shows up in a phylogenetic tree in Figure 1 within the WN02 clade which would be consistent with a post-2002 collection date, and as far as I can tell, the paper doesn't mention a pre-1999 sample in the USA:

* Amore, G., Bertolotti, L., Hamer, G.L., Kitron, U.D., Walker, E.D., Ruiz, M.O., Brawn, J.D. and Goldberg, T.L., 2010. [Multi-year evolutionary dynamics of West Nile virus in suburban Chicago, USA, 2005–2007](https://www.researchgate.net/publication/44607902_Multi-year_evolutionary_dynamics_of_West_Nile_virus_in_suburban_Chicago_USA_2005-2007). _Philosophical Transactions of the Royal Society B: Biological Sciences_, _365_(1548), pp.1871-1878.

I also tried searching earlier publications:

* Bertolotti, L., Kitron, U.D., Walker, E.D., Ruiz, M.O., Brawn, J.D., Loss, S.R., Hamer, G.L. and Goldberg, T.L., 2008. [Fine-scale genetic variation and evolution of West Nile Virus in a transmission “hot spot” in suburban Chicago, USA.](https://www.sciencedirect.com/science/article/pii/S0042682208000068) _Virology_, _374_(2), pp.381-389.
* Bertolotti, L., Kitron, U. and Goldberg, T.L., 2007. [Diversity and evolution of West Nile virus in Illinois and the United States, 2002–2005.](https://www.sciencedirect.com/science/article/pii/S0042682206007847) _Virology_, _360_(1), pp.143-149.

### Suggested change

I was going to [manually annotate these to unknown date "XXXX-XX-XX"](https://github.com/nextstrain/WNV/commit/b3097febbdbea2e88a5dc69ec4845cbd9d90d1b2) to avoid potentially skewing downstream analysis. My best guess is these sequences should be between 2002-2007.

### Full list of affected sequences

* https://pathoplexus.org/seq/PP_000JSDD.3
* https://pathoplexus.org/seq/PP_000JSEB.3

**Comment by emily-smith1** [comment]
Thanks for flagging this. Have there been any attempts to contact the corresponding author yet?

**Comment by j23414** [comment]
Not yet! I usually email NCBI helpdesk for fixing genbank annotations, and I'll probably try that next. Although I have a few other fixes to submit: [date, geoloc, and host fixes from annotations.tsv](https://github.com/nextstrain/WNV/blob/3556b10a378405f8a3f7976fdedc3afed29ff56e/ingest/defaults/annotations.tsv#L293-L306)

I imagine pathoplexus is still designing how to propagate community curated fixes back up to their NCBI records

**Comment by m-a-martin** [comment]
I agree with Jennifer and Emily that these collection dates are almost certainly spurious and that the correct collection dates are currently unknown. I do not think Pathoplexus currently supports annotating unknown collection dates as XXX-XX-XX. I believe Collection Date is currently a required field, so maybe these sequences should be removed from the database?
